roles/setup-ssl/tasks/main.yml

```
@@ -47,23 +49,18 @@
         -nodes
         -extensions v3_ca
         -days {{ item.days|default(3650) }}
-        -subj "/C={{ item.country|default('') }}/ST={{ item.state|default('') }}/L={{ item.city|default('') }}/O={{ item.organization|default('') }}/OU={{ item.unit|default('') }}{% if item.domains is defined %}{% for domain in item.domains %}/CN={{ domain }}{% endfor %}{% else %}/CN={{ item.name }}{% endif %}/emailAddress={{ item.email|default('') }}"
-        -keyout {{ SSL_LOCATION }}/private/{{ item.name }}.key
-        -out {{ SSL_LOCATION }}/certs/{{ item.name }}.crt
+        -subj "/C={{ item.country }}/ST={{ item.state }}/L={{ item.city }}/O={{ item.organization }}/OU={{ item.unit }}/emailAddress={{ item.email }}"
```

I noticed the for loop wasn't actually populating values so I removed it. As a small point, i think we should get out of the habit of using defaults to cover up errors. /shrug